new-app/src/App/runGame/RandNum.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import * as Number from '../../model/number';
+
+const RandomNumber = () => {
+
+  const getRandomDigit = (): Number.NumberDigit => {
```

Comment:
How about `buildRandomDigit` here? We could pull this out into the `Number` module and do `Number.buildRandomNumberDigit()`.